app/models/coronavirus/timeline_entry.rb

```diff
@@ -1,4 +1,7 @@
 class Coronavirus::TimelineEntry < ApplicationRecord
+  self.table_name = "coronavirus_timeline_entries"
+
+  belongs_to :coronavirus_page
```

Comment:
Should this be `belongs_to :page`?

Reply:
Yes it should, looks like a rebasing mistake but it's a bit of a worry that it all still built fine with that. Thanks for catching.